### DISCUSSION TITLE
BORIS topic on thebehaviourforum.org

### DISCUSSION THREAD

**Discussion by dhuzard** [olivierfriard/BORIS#632] [General]
A little bit of advertising, which might be of interest for you and other behavioralists ;) : we recently launched a forum for behavioral scientists that want to share/discuss different aspects of behavior ! here is the link: [https://www.thebehaviourforum.org/](https://www.thebehaviourforum.org/)

The reason I post it here is that I created a short topic about BORIS: [HERE](https://www.thebehaviourforum.org/t/boris-behavioral-observation-research-interactive-software/523)
and I thought you could be interested into participating and improving it, since you are the true expert !

Best,
Damien

**Comment by olivierfriard** [comment]
Thanks Damien!
but the site seems to be down...

**Reply by dhuzard** [reply]
Opps, there is indeed an issue with some web browsers: this one should work (I hope) www.thebehaviourforum.org

**Comment by dhuzard** [comment]
Hi again, there is a comment on the forum for which you might have a relevant answer: "[BORIS] is not appropriate if you want precise event durations as the reaction time to start and stop a new behavior is delayed."

Maybe, you could join and answer there directly ;)
Otherwise, no worries, I'll transmit you answer!
